schema/2.0.0/schema.md

```diff
@@ -643,20 +641,20 @@ Curators MUST annotate the following columns in the `var` and `raw.var` datafram
 </tbody></table>
 <br>
 
-### feature_id
+### index
 
 <table><tbody>
     <tr>
       <th>Key</th>
-      <td>feature_id (<code>var.index</code>/<code>raw.var.index</code>)</td>
+      <td>index</td>
     </tr>
     <tr>
       <th>Annotator</th>
       <td>Curator</td>
     </tr>
     <tr>
       <th>Value</th>
-        <td><code>str</code>. If the <code>feature_biotype</code> is <code>"gene"</code> then this MUST be an ENSEMBL term. If the <code>feature_biotype</code> is <code>"spike-in"</code> then this MUST be an ERCC Spike-In identifier. </td>
+        <td><code>str</code>. If the <code>feature_biotype</code> is <code>"gene"</code> then this MUST be an ENSEMBL term. If the <code>feature_biotype</code> is <code>"spike-in"</code> then this MUST be an ERCC Spike-In identifier.<br><br> <code>var.index</code> MUST contain unique identifiers for features. <code>raw.var.index</code> MUST be identical to <code>var.index</code>.<br><br></td>
```

Comment:
var.index MUST contain unique identifiers for features. _If AnnData.raw is present_, raw.var.index MUST be identical to var.index (?)

Reply:
...or maybe just one note at the intro to var and raw.var (Gene Metadata)...
"The requirements only apply to raw.var if AnnData.raw is present"

Reply:
I think that I prefer the second note.